test/wh_test_crypto.c

```diff
@@ -94,6 +95,16 @@ int whTest_CryptoClientConfig(whClientConfig* config)
     uint8_t authTag[16];
     uint8_t sharedOne[CURVE25519_KEYSIZE];
     uint8_t sharedTwo[CURVE25519_KEYSIZE];
+    uint8_t knownCmacKey[] = {0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6,
```

Comment:
Is this a NIST test vector?  Is so, add a comment

Reply:
I just pulled it out of wolfCrypt test, it didn't have any comments attached to it